src/surveys.py

```
@@ -0,0 +1,654 @@
+        headers={"Content-Type": "application/json", **CORS_HEADERS},
+    )
+
+
+def ok(data=None, msg: str = "OK"):
+    body = {"success": True, "message": msg}
+    if data is not None:
+        body["data"] = data
+    return json_resp(body, 200)
+
+
+def err(msg: str, status: int = 400, code: str = None):
```

Ruff (0.15.20)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)